deploy/spinnaker/complete/spinnakerservice.yml

```diff
@@ -11,8 +11,8 @@ spec:
       persistentStorage:
         persistentStoreType: s3
         s3:
-          bucket: mybucket
-          rootFolder: front50 # Change me
+          bucket: change-me # Change to a unique name. Spinnaker stores application and pipeline definitions here
```

Comment:
The Docs team is moving to angle brackets for [placeholders](https://kubernetes.io/docs/contribute/style/style-guide/#use-angle-brackets-for-placeholders).  So how about `<change-me>`... or `<globally-unique-bucket-name>`